OWNERS

```diff
@@ -1,7 +1,13 @@
 approvers:
+  - mheon
+  - baude
   - mrunalp
-  - runcom
   - rhatdan
+reviewers:
+  - mtrmac
+  - jwhonce
+  - runcom
   - nalind
-  - mheon
-  - baude
+  - wking
```

Comment:
From [the OWNERS docs][1]:

> All users are expected to be assignable. In GitHub terms, this means they are either collaborators of the repo, or members of the organization to which the repo belongs.

I don't think I'm either of those, although I'm fine becoming either.  That said, it may not matter until we have [tooling that is respecting the content of `OWNERS`][2], and in that case, I'm fine being listed here despite not being assignable.

[1]: https://github.com/kubernetes/community/blob/0a614ded6f114487dd78b5cec4329d19df8711d4/contributors/guide/owners.md#owners
[2]: https://github.com/projectatomic/libpod/issues/812#issuecomment-390484486

Reply:
Ok, I will add for now, we can add the proper plumbing later.